docs/Compliance.md

```diff
@@ -0,0 +1,124 @@
+# ATO and compliance considerations
+
+Every federal information system must be granted an Authority To Operate (ATO)
+by an Authorizing Official in order to go into production.  A good overview of
+the process can be found in this [excellent ATO overview](https://docs.google.com/presentation/d/1x-Bt8uyW-szHarglY57fFcC6PRCpUGFLe07uOvt93uk/edit#slide=id.g1f710bd7ce_0_847)
+and a more in-depth look can be found in [18F's before-you-ship docs](https://before-you-ship.18f.gov/ato/).  There
+are currently a few options for getting an ATO:
+* A [FedRAMP Tailored](https://tailored.fedramp.gov/) package.
+* A full [FedRAMP JAB ATO](https://www.fedramp.gov/jab-authorization/).
+* A GSA LATO (Lightweight ATO), which can be found by searching for "Lightweight
+  Security Authorization Guide" on 
+  [this page](https://insite.gsa.gov/topics/information-technology/security-and-privacy/it-security/it-security-procedural-guides)
+* Other agencies may have their own ATO process.
+
+What ATO you get depends on what your project needs.  If you are doing this in
+the GSA, you will probably want to follow the GSA LATO process, and most of the
+compliance data that we have collected here is aimed at fulfilling that process.
+That said, all of these ATO processes all map back to the NIST 800-53 controls,
+so if you decide on a different ATO package, you ought to be able to use the
+GSA LATO compliance data to help speed your ATO journey along.
+
+## Compliance Masonry
+
+We are using [Compliance Masonry](https://github.com/opencontrol/compliance-masonry) to
+document all of the controls and how the different components satisfy them.  Every
+component has a `compliance` directory which contains the documentation for that
+component.  For example, the `upload` app has a
+`Tanf-app/upload/compliance/component.yaml`
+file, and the whole project has a `Tanf-app/compliance` directory
+where everything is tied together with an `opencontrol.yaml` file, which
+documents all of the components, the certifications (GSA LATO), and the
```

Comment:
are we certain that the LATO controls are the control sets we need for HHS?

Reply:
No, we have no formal ATO package from HHS yet.  So the LATO stuff is kinda an easy placeholder.  I'm willing to bet that their control set is probably the entire 800-53 low set, from the random things that we've seen so far, but we don't know for sure.